tests/App.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
   <appSettings>
-    <add key="UATestEndpoint" value="opc.tcp://localhost:61210/UA/SampleClient"/>
+    <add key="UATestEndpoint" value="opc.tcp://md1m8frc:51210/UA/SampleServer"/>
```

Comment:
Don't want this change.
Comment to myself: This file probably should not be tracked to begin with. Should make a configurable template.